AppreciationApp/Repository/FithteenFiveAppreciationRepository.cs

```diff
@@ -31,40 +33,24 @@ public List<HighFives> GetWeeklyHighFives()
                 var highFives = new List<HighFives>();
 
                 //Initialise list of recipients
-                List<string> recipientsList = new List<string>();         
+                List<string> recipientsList = new List<string>();
+
 
                 foreach (var highFive in result.results)
                 {
-                    //IEnumerable<string>
-                    //var numberOfRecipients = highFive.text.Count(x => x.Contains("@")).Cast(IEnumerable<string>);
-                    //var recipientDefiner = "@";
-
-                    //Convert highFive text to manage-able string
-                    var text = highFive.text.ToString();
-                    dynamic textSplit;
-                    while (text.Contains("@"))
-                    {
-                        //Split text to find recipient
-                        textSplit = text.Split(new[] { '@', ' ' }, 3);
-                        text = textSplit[2];
-                        var recipient = textSplit[1].ToString();
-                        //Add recipient to list
-                        if (!string.IsNullOrEmpty(recipient))
-                        {
-                            recipientsList.Add(recipient);
-                        }                        
-                    }
+                    var highFiveMessage = highFive.text.ToString();
+                    dynamic textSplit = "";
 
-                    var receivers = "Welldone to: ";
-                    foreach (var receiver in recipientsList)
-                    {
-                        receivers = receivers + receiver + ", ";
+                    var copyOfHighFive = highFiveMessage;
```

Comment:
What is copyOfHighFive used for?

Reply:
Ah, it was from my previous working put - No longer needed - I'll remove it and push back up